ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/logging.j2

```diff
@@ -47,8 +47,19 @@ logging synchronous level {{ logging.synchronous.level | arista.avd.default("cri
 {{ logging_host_cli }}
 {%         endfor %}
 {%     endfor %}
-{%     if logging.format.timestamp is arista.avd.defined('high-resolution') %}
-logging format timestamp high-resolution
+{%     if logging.format.timestamp is arista.avd.defined %}
+{%         if logging.format.timestamp is arista.avd.defined('high-resolution') %}
+{%             set timestamp_cli = "logging format timestamp high-resolution" %}
+{%         elif "traditional" in logging.format.timestamp %}
```

Comment:
I think all this could be simplified a great deal, by just outputting `logging format timestamp {{ logging.format.timestamp }}` directly. Same for documentation.
The docs can then show all the possible combinations.

Reply:
@ClausHolbechArista would you keep the cases and just change how the output is generated. Or remove the cases and just check if timestamp is defined and if yes, render that?

Reply:
The latter. Just render if the var is defined